pages/common/rsstail.md

```diff
@@ -0,0 +1,23 @@
+# rsstail
+
+> Basically tail for RSS feeds.
```

Comment:
We prefer a formal language in our pages. Let us remove "Basically". And put tail in backticks.